### ISSUE TITLE
Output bundle on build

### ISSUE THREAD

**Issue by rreusser** [rreusser/plotly-mock-viewer#9]
This one is a little tricky. Should probably be optional since it means forking the bundle output stream both to the client and to the fs. Without buffering, I think that means getting into some stream details.

**Comment by rreusser** [comment]
Gave a shot at separating out the plotly bundler, and long story short, though I did my best to get thing browserifying as quickly and in parallel as possible, this doubles the browserify time from about 3-4 to 7-8 seconds, which I consider a blocker. Marking this `wontfix` so that you'll have to start `npm start watch` separately. 😞
